src/homework/day8/Numbers.java

```diff
@@ -5,47 +5,32 @@
 import java.util.Collections;
 import java.util.List;
 
-public class Numbers
-{
-    public static <T> List<T> reverseList(List<T> numbers)
-    {
-        List<T> reverse = new ArrayList<>(numbers);
-        Collections.reverse(reverse);
-        return reverse;
-    }
-
-    public static void main(String[] args)
-    {
+public class Numbers {
+    public static void main(String[] args) {
         Integer myNumb[] = {3342, 34, 79, 23426, 68, 1324, 55, 7699};
         List<Integer> numbers = Arrays.asList(myNumb);
 
```

Comment:
It's not necessary to create array myNumb[], you can fill it in an Arrays.asList